qchem/pennylane_qchem/qchem/hf/basis_set.py

```diff
@@ -87,23 +87,29 @@ def atom_basis_data(name, atom):
     >>> print(params)
     [((0, 0, 0), [3.425250914, 0.6239137298, 0.168855404], [0.1543289673, 0.5353281423, 0.4446345422])]
     """
-    basis_sets = {"sto-3g": STO3G}
+    basis_sets = {"sto-3g": STO3G, "6-31g": POPLE631G}
 
     s = [(0, 0, 0)]
     p = [(1, 0, 0), (0, 1, 0), (0, 0, 1)]  # for px, py, pz, respectively
 
     basis = basis_sets[name][atom]
     params = []
+    sp_count = 0
     for i, j in enumerate(basis["orbitals"]):
         if j == "S":
             params.append((s[0], basis["exponents"][i], basis["coefficients"][i]))
-        elif j == "SP":
+        if j == "SP":
```

Comment:
We will need more `if`s in the future, to handle p, d , ... orbitals, so I just got rid of the `elif`.

Reply:
I hope there should be an easier and more effective way than using multiple `if`s, in terms of maintainability of the codebase.

Reply:
At most, there are S, P, D, F, G, H orbitals for large basis sets and large atoms, e.g., Hg with cc-pVQZ-PP, and the data creation is not expensive. Not sure if there is a better way than using ~5-6 `if`s.

Reply:
Yeah, I wouldn't worry about this